napari/_qt/qt_main_window.py

```diff
@@ -924,8 +919,8 @@ def add_dock_widget(
         widget: QWidget,
         *,
         name: str = '',
-        area: str = 'bottom',
-        allowed_areas=None,
+        area: str = 'right',
```

Comment:
Should we make an `Enum` out of this 🤔 (not a blocker of this PR)

Reply:
yes.  later :)